index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+</head>
+<body style=" background-color: #a5d3eb;">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="container">
+        <div class="row">
+            <main id="main" class="text-center">
+                <h1 id="title">Alejandro and Ali </h1>
+                  <p>The men who learn coding</p>
+                <div id="img-div"> 
+                     <img id="image" src="https://media.istockphoto.com/id/1226577467/fr/photo/bons-amis-ensemble-appr%C3%A9ciant-la-vue-de-coucher-du-soleil.jpg?s=612x612&w=0&k=20&c=nRSV8e1my8nSozuQWKrOSxxKq6NAa8lJGlDi16jb_78=">
+                      <div id="img-caption">frindship is a situation in which two people are friends, we are freinds.</div> </div>
+                     <p id="tribute-info">frindship is a situation in which two people are friends, A friendship is a relationship or state of being friends between two people who like each other. Their friendship has lasted more than sixty years.</p>
+                <div> 
+                </div>
+                <a id="tribute-link" href="https://www.collinsdictionary.com/es/diccionario/ingles/friendship" target="_blank">About friendship</a>
+
```

Comment:
It might be better not to let white lines

Reply:
When reviewing you need to be 100% sure when passing remarks and comments for changes required. So avoid using `might` here.
Moreover, also state `why he needs to change it with a solid reason`.

Reply:
- [ ] Great job following the previous reviewer's advice, however, please consider giving a better reason, as explained above, saying it is "best practice" is too vague. 🤓 🔧